src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -93,6 +93,9 @@ const activityFeedInlineError: Errors = {
         content: messages.activityFeedItemApiError
     }
 };
+
+const RESIN_INFOPANE_TAG = 'infopane';
```

Comment:
Wait this is adding a component. We aren't adding any resin tags other than target in UI elements.

Reply:
Is there a way we could allow for generic attributes in content preview that get passed in by EUA? Similar to how many BRUI components do?

Reply:
that won't help much since we already have the page labeled...

Reply:
We could pass it in the sidebar props and apply ...rest to the DOM element like BRUI does in many instances